backend/src/generate_embeddings/main.py

```diff
@@ -4,10 +4,10 @@
 import io
 
 from aws_lambda_powertools import Logger
-from langchain_community.document_loaders import PyPDFLoader
+from langchain_community.document_loaders import PyPDFLoader, PyPDFDirectoryLoader
 from langchain_community.vectorstores import FAISS
 from langchain_openai.embeddings import OpenAIEmbeddings
-
+from zipfile import ZipFile
```

Comment:
Is zipfile a built in python library or will it have to be installed?

Reply:
it's a built-in library for Python v2.3 and up

Reply:
Alright, LGTM